src/dashboard/DashboardPage.vue

```diff
@@ -174,6 +171,11 @@ export default {
   created() {
     this.refresh();
   },
+  beforeCreate() {
+    // These below are async, might not have run in time
+    initThankfulTeamCreator();
+    db.attributeGithubActivity();
```

Comment:
Maybe add .catch() to these to clarify asyncness.

Reply:
Nah